cue/data/gen.cue

```diff
@@ -0,0 +1,213 @@
+                timepicker?: {
+                    // Whether timepicker is collapsed or not.
+                    collapse: bool | *false
+                    // Whether timepicker is enabled or not.
+                    enable: bool | *true
+                    // Whether timepicker is visible or not.
+                    hidden: bool | *false
+                    // Selectable intervals for auto-refresh.
+                    refresh_intervals: [...string] | *["5s", "10s", "30s", "1m", "5m", "15m", "30m", "1h", "2h", "1d"]
+                }
+                // Templating.
+                templating?: list: [...{...}]
+                // Annotations.
+                annotations?: list: [...{
+                    builtIn: number | *0
+                    // Datasource to use for annotation.
+                    datasource: string
+                    // Whether annotation is enabled.
+                    enable?: bool | *true
+                    // Whether to hide annotation.
+                    hide?: bool | *false
+                    // Annotation icon color.
+                    iconColor?: string
+                    // Name of annotation.
+                    name?: string
+                    type: string | *"dashboard"
+                    // Query for annotation data.
+                    rawQuery?: string
+                    showIn:   number | *0
+                }]
+                // Auto-refresh interval.
+                refresh?: string
+                // Version of the JSON schema, incremented each time a Grafana update brings
+                // changes to said schema.
+                schemaVersion: number | *25
+                // Version of the dashboard, incremented each time the dashboard is updated.
+                version?: number
+                panels?: [...#Panel]
+
+                // Dashboard panels. Panels are canonically defined inline
+                // because they share a version timeline with the dashboard
+                // schema; they do not vary independently. We create a separate,
+                // synthetic Family to represent them in Go, for ease of generating
+                // e.g. JSON Schema.
+                #Panel: {
+                    // The panel plugin type id. 
+                    type: !=""
+
+                    // Internal - the exact major and minor versions of the panel plugin
+                    // schema. Hidden and therefore not a part of the data model, but
+                    // expected to be filled with panel plugin schema versions so that it's
+                    // possible to figure out which schema version matched on a successful
+                    // unification.
+                    // _pv: { maj: int, min: int }
+                    // The major and minor versions of the panel plugin for this schema.
+                    // TODO 2-tuple list instead of struct?
+                    panelSchema: { maj: number, min: number }
```

Comment:
The placement of schema version numbers into the schema itself is a very important, and also unresolved topic within the scuemata pattern right now. i've deferred on it because we can ignore it in this very early stage, but it's something we'll need to deal with quickly shortly post-merge.

For now, consider this basically just a placeholder.